bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -385,6 +386,14 @@ spec:
           - patch
           - update
           - watch
+        - apiGroups:
+          - ""
+          resources:
+          - serviceaccounts
+          verbs:
+          - create
+          - delete
```

Comment:
I have mixed feelings about granting create and delete for serviceaccounts, an attacker could create or remove critical accounts, how to prevent being exploited?

Reply:
Any suggestions on how to handle this better?
I could create a different role that is namespace scoped instead?
config/rbac/wmco_serviceaccount_role.yaml
kind: Role 
namespace: system
resources: ["serviceaccounts"]
verbs: ["create", "delete", "get", "list"]

Reply:
or add application level security where we make sure the request is coming from valid account
if !strings.HasPrefix(roleName, "windows-instance-config-daemon-") {
	return fmt.Errorf("security violation: attempted to delete ClusterRole with invalid name pattern: %s", roleName)
	}